sycl/plugins/unified_runtime/pi_unified_runtime.cpp

```diff
@@ -910,6 +910,18 @@ __SYCL_EXPORT pi_result piSamplerRetain(pi_sampler Sampler) {
 __SYCL_EXPORT pi_result piSamplerRelease(pi_sampler Sampler) {
   return pi2ur::piSamplerRelease(Sampler);
 }
+/* clang-format off */

```

Comment:
there is 2 CI jobs that use different clang-format versions and they disagree on this part, so to pass the job, I have formatted with one of them and stopped clang-format from checking this part as a temporary fix for this.

Reply:
The 2 jobs are `Check code formatting / code_formatter (pull_request_target)` - `build / Build + LIT` -> `check-clang`

Reply:
got more info on this, refer to this [comment](https://github.com/intel/llvm/pull/12199#issuecomment-1874228112)